File: JAVA PROGRAMMING/Test25.java
// 25. PRINT SUM OF ALL EVEN NUMBER BETWEEN 0 TO N
import java.util.Scanner;
public class Test25 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();
        int sum = 0;
        for(int i = 0; i <= range; i++) {
            if(i % 2 == 0) {
                sum += i;
            }
        }
        System.out.println("Sum Of All Even Number From 0 to "+range+" is: "+sum);
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test25.ts
// 25. PRINT SUM OF ALL EVEN NUMBER BETWEEN 0 TO N
const func25 = (range: number): void => {
    let sum: number = 0;
    for(let i = 0; i <= range; i++) {
        if(i % 2 === 0) {
            sum += i;
        }
    }
    console.log(`Sum: ${sum}`);
}
func25(10);
func25(50);
func25(100);